src/GrumPHP/Configuration/GrumPHP.php

```diff
@@ -119,4 +120,12 @@ public function isBlockingTask($taskName)
         $taskMetadata = $this->getTaskMetadata($taskName);
         return $taskMetadata['blocking'];
     }
+
+    /**
+     * @return Filesystem
+     */
+    public function getFilesystem()
```

Comment:
Injecting the filesystem directly into the task now. Should I leave this method here and make a spec for it or remove it?